### PR TITLE
Gradle - Spring Boot Api - Attendance | Braulio de Jesús Luna Ocampo

### PR DESCRIPTION
**Pull request by jesus-luna**
# Gradle 

- Currency Format
- Convert Int To String
 # Spring Boot

- Pattern Syntax Checker
- Palindrome
- Anagram

### CODE REVIEW

File: jesus-luna/challenges/hacker-rank-solutions/src/main/java/bootcamp/hackerrank/august31/currency/CurrencyFormatter.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	public void formatCurrency(Double payment) {		
+		String[] isoCountries = Locale.getISOCountries();
+		Map<String, String> countriesMap = Arrays.asList(isoCountries)
+				.stream()
+				.map(isoCountry -> new Locale("", isoCountry))
+				.collect(Collectors.toMap(Locale::getDisplayCountry,Locale::getCountry));
+		
+		NumberFormat numberFormatIndia = NumberFormat.getCurrencyInstance(new Locale("en", countriesMap.get("India")));
+		NumberFormat numberFormatUsa = NumberFormat.getCurrencyInstance(Locale.US);
+    NumberFormat numberFormatChinese = NumberFormat.getCurrencyInstance(Locale.CHINA);
```
Comment (globant-bootcamp) [inline jesus-luna/challenges/hacker-rank-solutions/src/main/java/bootcamp/hackerrank/august31/currency/CurrencyFormatter.java:44]: fix indentation

Comment (globant-bootcamp) [reply jesus-luna/challenges/hacker-rank-solutions/src/main/java/bootcamp/hackerrank/august31/currency/CurrencyFormatter.java:44]: All the lines

**Review by globant-bootcamp — CHANGES_REQUESTED** [review]

**Review by jesus-luna — COMMENTED** [review]
Identation fixed

File: jesus-luna/challenges/hacker-rank-solutions/src/main/java/bootcamp/hackerrank/august31/currency/CurrencyFormatter.java (hunk 1 above)
Comment (jesus-luna) [reply jesus-luna/challenges/hacker-rank-solutions/src/main/java/bootcamp/hackerrank/august31/currency/CurrencyFormatter.java]: Done

**Review by jesus-luna — COMMENTED** [review]
indentation fixed correctly

**Pull request merged**